application/models/bestitamazonpay4oxidaddressutil.php

```diff
@@ -92,6 +92,11 @@ protected function _parseAddressFields($oAmazonData, array &$aResult)
      */
     public function parseAmazonAddress($oAmazonData)
     {
+        $this->getLogger()->debug(
+            'Amazon raw address',
+            array('amazonAddress' => $oAmazonData)
```

Comment:
Why the long syntax for arrays? It is not best it style and didn't oxid changed it as well?